content/tokio/tutorial/hello-tokio.md

```diff
@@ -189,15 +189,16 @@ The return value of an `async fn` is an anonymous type that implements the
 
 ## Async `main` function
 
-The main function used to launch the application differs from the usual Rust crate.
+The main function used to launch the application differs from the usual one
+found in most of Rust's crates.
 
 1. It is an `async fn`
 2. It is annotated with `#[tokio::main]`
 
 An `async fn` is used as we want to enter an asynchronous context. However,
 asynchronous functions must be executed by a [runtime]. The runtime contains the
 asynchronous task scheduler, provides evented I/O, timers, etc. The runtime does
-not automatically start, so the main function needs to start it.
+not automatically start, thus the main function needs to start it.
```

Comment:
I think the use of "thus" here is actually less correct than "so".

Reply:
@hawkw you are right!
https://jakubmarian.com/so-thus-therefore-and-hence-in-english/